src/homework/api/user.py

```diff
@@ -1,3 +1,29 @@
-from fastapi import APIRouter
+from typing import Annotated
+from fastapi import APIRouter, Depends, HTTPException
+from .contracts import UserBalanceResponse, UserBalanceRequest, AccessLevel

```

Comment:
I suggest importing full path to avoid relative imports.

Reply:
I agree with you. It is better to be consistent in imports and use full paths